ra_aid/tools/programmer.py

```
@@ -1,13 +1,12 @@
 import os
-from typing import List, Optional, Dict, Union
+from typing import List, Dict, Union
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)